client/app/l10n/en.json

```diff
@@ -15,7 +15,7 @@
   "Access expiration": "Access expiration",
   "Access granted": "Access granted",
   "Access requested": "Access requested",
-  "Access to the whistleblower's identity has been requested to the custodian.": "Access to the whistleblower's identity has been requested to the custodian.",
+  "Access to the whistleblower's identity has been requested to the custodian.": "The custodian has requested to know who the whistleblower is.",
```

Comment:
Actually this corections modify the string: It is the recipient that is asking the access and this message clarify the recipient that it's request is pending.

Reply:
The string didn't make sense before, and now I don't know what is going on.
"The whistleblower has requested to know who the custodian is" ?